packages/toolkit/scripts/build/handlePackageJson.ts

```diff
@@ -41,7 +41,7 @@ function main() {
 
         // Remove Amazon Q extension settings stored in core
         Object.keys(coreSettings).forEach(key => {
-            if (key.startsWith('aws.amazonq') || key.startsWith('aws.codeWhisperer')) {
+            if (key.startsWith('aws.amazonQ') || key.startsWith('aws.codeWhisperer')) {
```

Comment:
Do we still need the `codeWhisperer` condition in here? - `|| key.startsWith('aws.codeWhisperer')` ?

Reply:
I double checked and there is no settings starting 'aws.codeWhisperer' anymore. I can remove it.